packages/form/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@development-framework/form",
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "src",
+    "tsconfig.json"
+  ],
+  "scripts": {
+    "prebuild": "rm -rf dist && yarn install",
+    "build": "tsc",
+    "postbuild": "cp package.json ./dist/ &&  rm -rf node_modules",
+    "test": "jest test",
+    "test-watch": "jest test --watchAll"
+  },
+  "dependencies": {
+    "object-path": "^0.11.8",
+    "react-hook-form": "^7.31.2"
+  },
+  "devDependencies": {
+    "@development-framework/dm-core": ">=1.0.25",
+    "@development-framework/tabs": ">=1.0.2",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/react-hooks": "^7.0.2",
+    "@testing-library/user-event": "14.4.3",
+    "@types/jest": "^27.4.0",
+    "@types/react": "^17.0.39",
+    "@types/react-dom": "^17.0.11",
+    "@types/react-router-dom": "^5.3.3",
+    "@types/styled-components": "^4.1.18",
+    "jest": "^27.1.3",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-router-dom": ">=5.1.2",
+    "react-test-renderer": "^17.0.2",
+    "styled-components": ">=5.3.6",
+    "ts-jest": "^27.1.3",
+    "ts-loader": "^9.3.1",
+    "typescript": "^4.5.5"
+  },
+  "peerDependencies": {
+    "@development-framework/dm-core": ">=1.0.25",
```

Comment:
should use a later version of dm-core, 1.0.34

Reply:
upgraded to >1.0.35